### ISSUE TITLE
Unable to open splits when focus in nvim-tree

### ISSUE THREAD

**Issue by petertriho** [kyazdani42/nvim-tree.lua#418]
https://github.com/kyazdani42/nvim-tree.lua/commit/be184bd94e558b302461b359d02ed1b01f7154a1 seems to have broken the ability to open splits when I'm focused in nvim-tree i.e. I use to be able to run :Git (from vim-fugitive) or :Neogit kind=split (from neogit) to open the git status split

**Comment by rekaerst** [comment]
be184bd also causes nvim-tree to clone itself when opening a new window while focused in nvim-tree.
![image](https://user-images.githubusercontent.com/52642831/120210242-28154c00-c262-11eb-9df9-c62d68415649.png)

And messed up floating window.
![image](https://user-images.githubusercontent.com/52642831/120209497-57778900-c261-11eb-9963-15cfe9e4eeef.png)

```lua
function M.save_previous_buf()
  vim.cmd "wincmd p"
  old_win = api.nvim_get_current_win()
  vim.cmd "wincmd p"
end

function M.set_previous_buf()
  if old_win then
    api.nvim_set_current_win(old_win)
  end
end
```

**Comment by kyazdani42** [comment]
sorry for the inconvenience... a mistake on my end. Fixed by #421

**Comment by petertriho** [comment]
@kyazdani42 No worries! Thanks you so much for the awesome plugin
